test/utils/string.formatter.spec.unit.ts

```diff
@@ -0,0 +1,55 @@
+import { removeNonPrintableChars } from "../../src/utils/string.formatter";
+
+
+describe("String formatter tests", () => {
+  it("should not remove formatting chars", () => {
+    const dodgyString: string = "hello.\r\nThis string is not dodgy!";
```

Comment:
perhaps rename this (for all the tests) to preFormattedString

Reply:
I like "dodgyString"

Reply:
but this example, it is not a dodgy string

Reply:
Renamed these.